pipelines/template-build/template-build.yaml

```diff
@@ -241,6 +241,19 @@ spec:
       workspaces:
       - name: workspace
         workspace: workspace
+    - name: fbc-validation
```

Comment:
I feel we don't need to add this step to `template-build.yaml` since it is not needed by all pipelines, but add it to add to `pipelines/fbc-builder/patch.yaml` as https://github.com/redhat-appstudio/build-definitions/pull/249 does

Reply:
Good catch, once the #249 get merged, I will update the PR.

Reply:
added